resources/lib/ui/utils.py

```diff
@@ -13,6 +14,9 @@ def fetch_sources(sources, dialog, raise_exceptions=False):
     fetched_sources = []
```

Comment:
This setting can be then read from deafult.py (PLAY) and then pass 0 or 1 to SourcesList class which will pass it to fetch_sources when called.